tests/eas/load_tracking.py

```diff
@@ -172,6 +175,81 @@ def get_signal_mean(self, experiment, signal,
         signal = select_window(signal, window)
         return area_under_curve(signal) / (window[1] - window[0])
 
+    def get_task_start_time(self, experiment, task):
+        """
+        Get the instant of time when the task starts running for the very first
+        time
+
+        :param experiment: Experiment to get trace for
+
+        :param task: Name of the task
+        :type task: str
+        """
+        trace = self.get_trace(experiment)
+        if 'sched_switch' not in trace.available_events:
+            raise ValueError('No sched_switch events. '
+                             'Does the kernel support them?')
+
+        # Get instant of time when the task starts running
+        sw_df = trace.ftrace.sched_switch.data_frame
+        pid = sw_df[sw_df.prev_comm == task].prev_pid.iloc[0]
+        return sw_df[sw_df.next_pid == pid].index[0]
```

Comment:
`LisaTest` has a `get_start_time` method - can you integrate this with the logic used there, and put it next to it?